resources/language/resource.language.nl_nl/strings.po

```diff
@@ -0,0 +1,166 @@
+"Project-Id-Version: service.upnext\n"
+"Report-Msgid-Bugs-To: http://trac.xbmc.org/\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2018-02-26 13:31+0000\n"
+"Last-Translator: dagwieers\n"
+"Language-Team: Dutch\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: nl\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.7\n"
+
+msgctxt "#30001"
+msgid "The number of seconds before the end to show the notification"
+msgstr "Aantal seconden voor het einde om meldingen te tonen"
+
+msgctxt "#30002"
+msgid "Log Level:"
+msgstr "Log niveau"
+
+msgctxt "#30003"
+msgid "Settings"
+msgstr "Instellingen"
+
+msgctxt "#30004"
+msgid "Advanced"
+msgstr "Geavanceerd"
+
+msgctxt "#30005"
+msgid "Default action when nothing selected"
+msgstr "Standaard actie als niets geselecteerd is"
+
+msgctxt "#30006"
+msgid "Watch Now"
+msgstr "Bekijk nu"
+
+msgctxt "#30008"
+msgid "Up Next"
+msgstr "Volgend"
+
+msgctxt "#30009"
+msgid "Number of episodes before still watching query"
+msgstr "Aantal afleveringen alvorens te vragen of er nog gekeken wordt"
+
+msgctxt "#30010"
+msgid "Continue watching"
+msgstr "Verder kijken"
+
+msgctxt "#30013"
+msgid "Include watched episodes"
+msgstr "Toepassen op bekeken afleveringen"
+
+msgctxt "#30014"
+msgid "Disable Up Next (restart required)"
+msgstr "Schakel Up Next uit (herstarten vereist)"
+
+msgctxt "#30017"
+msgid "Automatically play short videos"
+msgstr "Speel korte video's automatisch af"
+
+msgctxt "#30018"
+msgid "Maximum length for automatic short video play (in minutes)"
+msgstr "Maximale lengte voor het automatisch afspelen van korte video's (in minuten)"
+
+msgctxt "#30019"
+msgid "Show on-screen notification during short video autoplay"
+msgstr "Toon meldingen tijdens het automatisch afspelen van korte video's"
+
+msgctxt "#30024"
+msgid "Still there?"
+msgstr "Ben je er nog?"
+
+msgctxt "#30025"
+msgid "Set the display mode for the notifications"
+msgstr "Schermmodus voor meldingen"
+
+msgctxt "#30028"
+msgid "Developer"
+msgstr "Ontwikkelaar"
+
+msgctxt "#30029"
+msgid "Enable developer mode"
+msgstr "Schakel ontwikkelaarmodus in"
+
+msgctxt "#30030"
+msgid "Choose the window to be shown"
+msgstr "Kies het venster om te tonen"
+
+msgctxt "#30032"
+msgid "Enable on playlists"
+msgstr "Toepassen op afspeellijsten"
+
+msgctxt "#30034"
+msgid "Close"
+msgstr "Sluit"
+
+msgctxt "#30035"
+msgid "seconds"
+msgstr "seconden"
+
+msgctxt "#30036"
+msgid "in"
+msgstr "in"
+
+msgctxt "#30037"
+msgid "Next Episode in"
+msgstr "Volgend aflevering in"
+
+msgctxt "#30038"
+msgid "Simple"
+msgstr "Eenvoudig"
+
+msgctxt "#30039"
+msgid "Fancy"
+msgstr "Mooi"
+
+msgctxt "#30040"
+msgid "Play"
+msgstr "Afspelen"
+
+msgctxt "#30041"
+msgid "Don't Play"
+msgstr "Niet afspelen"
+
+msgctxt "#30042"
+msgid "None"
+msgstr "Geen"
+
+msgctxt "#30043"
+msgid "Info"
+msgstr "Info"
+
+msgctxt "#30044"
+msgid "Debug"
+msgstr "Debug"
```

Comment:
Foutopsporing

Reply:
Hier gaat het over het log level, en ik ben niet zo zeker of het dan nog duidelijk is.
In de bestaande Kodi add-ons (slechts twee) wordt het telkens niet vertaald.
Maar ik ga het toch doen.

Reply:
Fixed.